File: app/src/main/java/com/example/hades/androidpo/_1_render_op/startup/v1/RxLifeCycleAndroidBinderImpl.java
package com.example.hades.androidpo._1_render_op.startup.v1;

import io.reactivex.Scheduler;
import io.reactivex.android.schedulers.AndroidSchedulers;

public class RxLifeCycleAndroidBinderImpl extends RxLifeCycleSchedulerBinderImpl {
    public RxLifeCycleAndroidBinderImpl() {
        this(AndroidSchedulers.mainThread());
    }

    public RxLifeCycleAndroidBinderImpl(Scheduler scheduler) {
        super(scheduler);
    }
}

File: app/src/main/java/com/example/hades/androidpo/_1_render_op/startup/v2/RxLifeCycleAndroidBinderImpl.kt
package com.example.hades.androidpo._1_render_op.startup.v2

import io.reactivex.Scheduler
import io.reactivex.android.schedulers.AndroidSchedulers

class RxLifeCycleAndroidBinderImpl @JvmOverloads constructor(scheduler: Scheduler = AndroidSchedulers.mainThread()) :
    RxLifeCycleSchedulerBinderImpl(scheduler)